integration-tests/testutils/common/common.go

```diff
@@ -118,13 +119,20 @@ func (s *SnappySuite) SetUpTest(c *check.C) {
 	}
 }
 
+// OSSnapName returns the name of the OS snap.
+func OSSnapName(c *check.C) string {
+	snappyOS, err := partition.SnappyOS()
+	c.Assert(err, check.IsNil, check.Commentf("Error getting the name of the OS snap: %s", err))
+	return strings.Split(snappyOS, ".")[0]
+}
+
```

Comment:
I prefer to move things from common to the proper package instead of adding more code to it, what do you think? Would the testutils/bootloader package be a good place for this function? Or perhaps a new one, including GetCurrentUbuntuCoreVersion?